content/main-config.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Change file name to `mainConfig.json`

Because we never used `main-config.json`